core/block/editor/state/change.go

```diff
@@ -759,6 +781,42 @@ func (s *State) makeOriginalCreatedChanges() (ch []*pb.ChangeContent) {
 	return
 }
 
+func (s *State) makeNotificationChanges() []*pb.ChangeContent {
+	var changes []*pb.ChangeContent
+	if s.parent == nil || s.parent.notifications == nil {
```

Comment:
I think you should use getter for `notifications`, because parent's state could not have initialized `notifications`. Maybe I'm wrong

Reply:
Fixed